src/contact/parallel/bvh_contact_manager.cc

```diff
@@ -65,6 +65,50 @@
 
 namespace nimble {
 
+namespace details {
+
+#ifdef NIMBLE_HAVE_ARBORX
+struct ArborXCallback
+{
+  nimble_kokkos::DeviceContactEntityArrayView d_contact_faces;
+  nimble_kokkos::DeviceContactEntityArrayView d_contact_nodes;
+
+  double enforcement_penalty;
+
+  std::vector<NarrowphaseResult> &resa_vec;
+  std::vector<NarrowphaseResult> &resb_vec;
+
+  template <typename Query>
+  KOKKOS_FUNCTION void
+  operator()(Query const& query, int j) const
+  {
+    auto& myNode = d_contact_nodes(ArborX::getData(query));
+    auto& myFace = d_contact_faces(j);
+
+    bool   inside               = false;
+    double normal[3] = {0.0, 0.0, 0.0};
+
+    //--- Determine whether the node is projected inside the triangular face
+    //--- Determine whether the node is projected inside the triangular face
```

Comment:
duplicate comment?

Reply:
@rjones-sandia 

You are right. Bad copy and paste.
I will remove it on a different PR (to avoid a new round of approval).